model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -174,9 +249,11 @@ def __init__(
             }
         )
         self.register_provider(input_fields_provider)
-        self._register_computed_fields()
+        self._register_computed_fields(input_fields_provider)
 
-    def _register_computed_fields(self) -> None:
+    def _register_computed_fields(
+        self, input_fields_provider: factory.PrecomputedFieldProvider
```

Comment:
```suggestion
        self
```
this input is unused?